test/interval_tests/numeric_tests.jl

```diff
@@ -73,9 +73,9 @@ facts("Numeric tests") do
     @fact @biginterval(-3,4) ^ 0.5 --> @biginterval(0, 2)
 
     @fact @interval(1,27)^@interval(1/3) --> roughly(Interval(1., 3.))
-    @fact @interval(1,27)^(1/3) --> Interval(1., 3.)
-    @fact @interval(1,27)^(1//3) --> Interval(1., 3.)
-    @fact @interval(0.1,0.7)^(1//3) --> Interval(0.46415888336127786, 0.8879040017426009)
+    @fact @interval(1,27)^(1/3) --> roughly(Interval(1., 3.))
```

Comment:
Why is this test change needed? Before we had the nice and correct behavior that `@fact @interval(1,27)^(1/3) --> Interval(1., 3.)` was `true`.